kokoro/bench.sh

```diff
@@ -50,13 +50,17 @@ function install_requirements() {
     echo Installing parquet demo requirements.
     pip install -r ./demo/lightning/text-based/distributed/requirements.txt
 
+    echo Installing image training demo requirements.
+    pip install -r ./demo/lightning/image-segmentation/requirements.txt
+
     echo Installing required dependencies.
     pip install .
 }
 
 function run_benchmarks(){
     echo Running benchmarks.
     python3 -u ./demo/lightning/text-based/distributed/model.py --local --project=dataflux-project --bucket=fineweb-df-benchmark --num-workers=2 --num-nodes=1 --devices=5 --batch-size=512 --epochs=5 --limit-train-batches=1000 --log-level=ERROR;
+    python3 -u ./demo/lightning/image-segmentation/train.py --local --gcp_project=dataflux-project --gcs_bucket=dataflux-demo-public --images_prefix=image-segmentation-dataset/images --labels_prefix=image-segmentation-dataset/labels --num_nodes=1 --num_devices=5 --epochs=5;
```

Comment:
Add another echo here to indicate what benchmark is running for the sake of easily checking the logs.

Reply:
We may even want to add timing logs in the future.